src/ducks/PomodoroSlice.js

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { initialSettings } from '../services/settings';
+
+const initialState = {
+  type: 'pomodoro',
+  time: 25,
+  isRunning: false,
+  start: 0,
+  config: initialSettings
+};
+
+const pomodoroSlice = createSlice({
+  name: 'pomodoro',
+  initialState,
+  reducers: {
+    setType: (state, action) => {
```

Comment:
I notice that you are calling 3 actions everytime you need a new timer, like in this code:
```javascript
  dispatch(setType(type));
  dispatch(setStart(0));
  dispatch(setRunning(false));
```
I think it would be better to have a single reducer (something like timerHandler) for this type of thing

Reply:
Done!